_includes/scenarios/integrate/data-model-overview.md

```diff
@@ -241,139 +241,257 @@ With the information in this type you can clearly identify the Transport object
 
 ### LocalAttributeShareInfo
 
-The Local Attribute Share Info helps to keep track of how the Local Attribute was received/sent, from whom it was received/who sent it, as well as which Local Attribute it was copied from (in case of a shared Repository Attribute). For example, this enables us to track back who we shared a certain Repository Attribute with, so we are able to notify each of them when changing the Repository Attribute.
+The LocalAttribute Share Info helps to keep track of how the LocalAttribute was received/sent, from whom it was received/who sent it, as well as which LocalAttribute it was copied from (in case of a shared Repository Attribute). For example, this enables us to track back who we shared a certain Repository Attribute with, so we are able to notify each of them when changing the Repository Attribute.
 
-| Name             | Type                    | Description                                                                                                                 |
-| ---------------- | ----------------------- | --------------------------------------------------------------------------------------------------------------------------- |
-| requestReference | `string`                | The ID of the Local Request the Local Attribute was received in/sent with.                                                  |
-| peer             | `string`                | The Address of the Identity the Local Attribute was received from/shared with.                                              |
-| sourceAttribute  | `string` \| `undefined` | If the Local Attribute is a copy of a Repository Attribute, then this property contains the ID of the Repository Attribute. |
+| Name             | Type                    | Description                                                                                                                |
+| ---------------- | ----------------------- | -------------------------------------------------------------------------------------------------------------------------- |
+| requestReference | `string`                | The ID of the Local Request the LocalAttribute was received in/sent with.                                                  |
+| peer             | `string`                | The Address of the Identity the LocalAttribute was received from/shared with.                                              |
+| sourceAttribute  | `string` \| `undefined` | If the LocalAttribute is a copy of a Repository Attribute, then this property contains the ID of the Repository Attribute. |
 
 ## LocalAttributeListener
 
 A LocalAttributeListener is created when you accept an incoming Request with a [`RegisterAttributeListenerRequestItem`]({% link _docs_integrate/requests-and-requestitems.md %}#registerattributelistenerrequestitem). It is used to keep track of which Attribute Listeners currently exist and what they are listening for.
```

Comment:
What about this? Do we have/use LocalAttributeListeners, if we don't use the RegisterAttributeListenerRequestItem?

Reply:
Erm, no. But why can't we use the RegisterAttributeListenerRequestItem?

Reply:
I was asking about it yesterday and it doesn't seem to be in use. Also, it was commented out in the documentation on the Requests and RequestItems page.

Reply:
We don't know anyone that is actively using it, but we offer the functionality so we should leave the documentation as is.

Reply:
Should I then also document it here (and on the Request and Response Introduction page)?

Reply:
@jkoenig134 This seems to be the one thing blocking this PR and it would be great to merge it soon, since many other PRs are dependent on the new links. If this question needs further discussion, can we postpone it to a different PR?

Reply:
yep, please leave anything as is for now.